debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/RowData.java

```diff
@@ -42,23 +41,17 @@ public boolean hasCell(String columnName) {
         return cellMap.containsKey(columnName);
     }
 
-    @Override
-    public GenericRecord record(Schema schema) {
-        GenericRecordBuilder builder = new GenericRecordBuilder(schema);
-        for (Schema.Field field : schema.getFields()) {
-            // note: calling getFieldSchema twice since the field has UNION type,
-            // and we only want to extract the non-null field schema in the UNION type.
-            // they currently have the same name
-            Schema unionSchema = getFieldSchema(field.name(), schema);
-            Schema cellSchema = getFieldSchema(field.name(), unionSchema);
```

Comment:
thanks for cleaning up this mess of mine!

Reply:
😆